modules/server/src/main/scala/navigate/server/tcs/TcsBaseControllerEpics.scala

```diff
@@ -382,7 +388,7 @@ class TcsBaseControllerEpics[F[_]: Async: Parallel](
 
     config.m2Guide match {
       case M2GuideConfig.M2GuideOff               =>
-        m1(tcsEpics.startCommand(timeout)).m2GuideCommand
+        m1(gains(tcsEpics.startCommand(timeout))).m2GuideCommand
```

Comment:
Is this the correct way to chain these calls?

Reply:
It is prettier if you use `.combine`